### PR TITLE
acpi: Reword PLIC/APLIC namespace requirement

### PR DESCRIPTION
**Pull request by vlsunil**
ACPI_80 requirement is only for systems with either PLIC or APLIC. Reword it to make it clear.

### CODE REVIEW

**Comment by vlsunil** [comment]
@ved-rivos @andreiw - please help review whether this makes requirement clear.

**Review by adurbin-rivos — APPROVED** [review]

**Review by ved-rivos — APPROVED** [review]

**Comment by ved-rivos** [comment]
@vlsunil could we also update this statement:
"the MADT will also contain either PLIC or IMSIC/APLIC structures"
to
"the MADT will also contain either PLIC or APLIC structures"

**Review by andreiw — APPROVED** [review]

**Comment by andreiw** [comment]
> @vlsunil could we also update this statement: "the MADT will also contain either PLIC or IMSIC/APLIC structures" to "the MADT will also contain either PLIC or APLIC structures"

Yep, doing so in https://github.com/riscv-non-isa/riscv-brs/pull/166
